tests/parent-child-node.html

```diff
@@ -0,0 +1,87 @@
+<!DocType HTML>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../node_modules/@webcomponents/webcomponents-platform/webcomponents-platform.js"></script>
+    <script src="../node_modules/@webcomponents/template/template.js"></script>
+    <script>
+      ShadyDOM = {force: true};
+    </script>
+    <!-- <script src="../shadydom.min.js"></script> -->
+
+    <script src="../node_modules/wct-browser-legacy/browser.js"></script>
+  </head>
+  <body>
+    <template>Text<!--Comment--><p>Element</p></template>
+
+    <div></div>
+
+    <script>
+      var childNodes,
+        fragment = document.querySelector('template').content.cloneNode( true ),
+        sandBox = document.querySelector('div').attachShadow({mode: 'open'});
+
+      function getNodeText() {
+
+        return  Array.prototype.map.call(sandBox.childNodes, function (node) {
+
+          return  node.nodeValue || node.textContent;
+        });
+      }
+
+      test('remove() of ChildNode interface',  function () {
+
+        childNodes = Array.apply(null, fragment.childNodes);
+
+        for (var i = 0;  childNodes[i];  i++)  childNodes[i].remove();
```

Comment:
Always add braces for `for` and `if`

Reply:
I think we should set a more strict **ESLint rule** to uniform code style of every coder.